.circleci/config.yml

```diff
@@ -0,0 +1,329 @@
+# CircleCI v2.1 Config
+version: 2.1
+
+##
+# defaults
+#
+# YAML defaults templates, in alphabetical order
+##
+defaults_Dependencies: &defaults_Dependencies |
+    apk --no-cache add git
+    apk --no-cache add ca-certificates
+    apk --no-cache add curl
+    apk --no-cache add openssh-client
+    apk --no-cache add bash
+    apk add --no-cache -t build-dependencies make gcc g++ python libtool autoconf automake
+    npm config set unsafe-perm true
+    npm install -g node-gyp
+
+defaults_Typescript: &defaults_Typescript |
```

Comment:
I'm not a big fan of these yaml templates, especially for a command so short and only used twice

Reply:
removed. template replaced with command.